compilator/compilator.go

```diff
@@ -88,21 +89,22 @@ func NewDockerCompilator(
 	keepContainer bool,
 	ui *termui.UI,
 	grapher util.ModelGrapher,
+	packStorage *PackageStorage,
```

Comment:
This should be `packageStorage` instead of `packStorage` for consistency with naming elsewhere.